packages/ssi-types/src/mapper/credential-mapper.ts

```diff
@@ -454,22 +451,23 @@ export class CredentialMapper {
   static toUniformCredential(
     verifiableCredential: OriginalVerifiableCredential,
     opts?: {
-      maxTimeSkewInMS?: number
+      maxTimeSkewInMS?: number,
+      hasher?: Hasher
     },
   ): IVerifiableCredential {
     if (CredentialMapper.isSdJwtDecodedCredential(verifiableCredential)) {
       throw new Error('Converting SD-JWT VC to uniform VC is not supported.')
     }
     const original =
-      typeof verifiableCredential !== 'string' && CredentialMapper.hasJWTProofType(verifiableCredential)
+      typeof verifiableCredential !== 'string' && CredentialMapper.hasJWTProofType(verifiableCredential) && !CredentialMapper.isSdJwtEncoded(verifiableCredential.proof.jwt)
```

Comment:
not sure if `verifiableCredential.proof.jwt` should be passed here

Reply:
Yes it should